Integrations/SlashNextPhishingIncidentResponse/SlashNextPhishingIncidentResponse.py

```diff
@@ -0,0 +1,1180 @@
+    :param scanid: Scan ID returned by a SlashNext API earlier as a result of a scan request
+    :param resolution: Desired resolution of the screenshot. Currently supported values are 'high' and 'medium'
+    :return: Response of the SlashNext download/screenshot API
+    """
+    # Create the required data dictionary for Download/Screenshot
+    api_data = {
+        'scanid': scanid,
+        'resolution': resolution
+    }
+    response = http_request(endpoint=DL_SC_API, data=api_data)
+
+    if response.get('errorNo') == 1:
+        demisto.results(
+            'Your Url Scan request is submitted to the cloud and may take up-to 60 seconds to complete.\n'
+            'Please check back later using "slashnext-download-screenshot" command with Scan ID = {}'.format(scanid))
+    elif response.get('errorNo') != 0:
+        demisto.results('API Returned, {}:{}!'.format(response.get('errorNo'), response.get('errorMsg')))
+
+    return response
+
+
+def download_screenshot_command():
+    """
+    Execute SlashNext's download/screenshot API against the already requested URL scan command with the given parameters
+    @:return: None
+    """
+    # 1. Get input scan id and resolution from Demisto
+    scanid = demisto.args().get('scanid')
+    resolution = demisto.args().get('resolution')
+    # 2. Get the forensic webpage screenshot from SlashNext API
+    response = download_screenshot(scanid=scanid, resolution=resolution)
+    if response.get('errorNo') != 0:
+        return
+    # 3. Parse and format the response
+    sc_base64 = response.get('scData').get('scBase64')
+    sc_data = base64.b64decode(sc_base64)
+
+    sc_file = fileResult('slashnext_{}.jpg'.format(scanid), sc_data, entryTypes['image'])
+
+    demisto.results({
+        'Type': entryTypes['image'],
+        'ContentsFormat': formats['text'],
+        'Contents': 'Forensics: Webpage Screenshot for URL Scan ID = {}'.format(scanid),
+        'File': sc_file.get('File'),
+        'FileID': sc_file.get('FileID')
+    })
+
+
+def download_html(scanid):
+    """
+    Execute SlashNext's download/html API against the already requested URL scan with the given parameters
+    :param scanid: Scan ID returned by a SlashNext API earlier as a result of a scan request
+    :return: Response of the SlashNext download/html API
+    """
+    # Create the required data dictionary for Download/HTML
+    api_data = {
+        'scanid': scanid
+    }
+    response = http_request(endpoint=DL_HTML_API, data=api_data)
+
+    if response.get('errorNo') == 1:
+        demisto.results(
+            'Your Url Scan request is submitted to the cloud and may take up-to 60 seconds to complete.\n'
+            'Please check back later using "slashnext-download-html" command with Scan ID = {}'.format(scanid))
+    elif response.get('errorNo') != 0:
+        demisto.results('API Returned, {}:{}!'.format(response.get('errorNo'), response.get('errorMsg')))
```

Comment:
Should be returned as an error
![image](https://user-images.githubusercontent.com/31018228/66460380-7703e180-ea7f-11e9-830c-8adc67d908fe.png)